instantsearch/src/main/java/com/algolia/instantsearch/helpers/Searcher.java

```diff
@@ -525,7 +525,7 @@ public void addFacet(@NonNull String attribute, boolean isDisjunctive, @Nullable
      * */
     @SuppressWarnings({"WeakerAccess", "unused"}) // For library users
     public Searcher setFacet(@NonNull String attribute, boolean isDisjunctive) {
-        if (isDisjunctive && disjunctiveFacets.contains(attribute)) {
+        if (isDisjunctive && !disjunctiveFacets.contains(attribute)) {
```

Comment:
Whoops, I should have catched that :sweat_smile:

Reply:
That's my bad.

Reply:
Noprob, I'll merge and publish an update.

Reply:
Noprob, I'll merge and publish an update. One request though, would you mind rebasing your commits to follow the [conventional changelog](http://conventionalcommits.org/) approach? This way the changelog gets generated and we don't have to manually edit it after the release. Something like:

- `chore(gradle): Use gradle 4.4 with android build tool 3.1.3` 
- `fix(Searcher): setFacet logic error`